clients/spark/core/src/test/scala/io/treeverse/jpebble/BlockParserSpec.scala

```diff
@@ -261,3 +266,134 @@ class CountedIteratorSpec extends AnyFunSpec with Matchers {
     }
   }
 }
+
+class GolangContainerSpec extends AnyFunSpec with ForAllTestContainer {
+
+  override val container: GenericContainer = GenericContainer("golang:1.16.2-alpine",
+    classpathResourceMapping = Seq(
+      ("parser-test/sst_files_generator.go", "/local/sst_files_generator.go", BindMode.READ_WRITE),
+      ("parser-test/go.mod", "/local/go.mod", BindMode.READ_WRITE),
+      ("parser-test/go.sum", "/local/go.sum", BindMode.READ_WRITE)),
+    command = Seq("/bin/sh", "-c", "cd /local && CGO_ENABLED=0 go run sst_files_generator.go && echo \"done\""),
+    waitStrategy = new LogMessageWaitStrategy().withRegEx("done\\n")
```

Comment:
I would _really_ like the comments @johnnyaug and I placed on this wait strategy to be resolved, or the reason to avoid them logged on this PR.  Why not use the [indefinite on-shot strategy](https://www.testcontainers.org/features/startup_and_waits/#indefinite-one-shot-startup-strategy-example)?  It is intended precisely for this option, and would simplify `command`, and is probably _safer_.

Reply:
Let's add a TODO and/or an issue to use the right strategy here!